data/Language/Pascal.yaml

```diff
@@ -72,8 +72,8 @@ keywords:
   - with
   - xor
 datatypes:
-  - string
-  - boolean
-  - integer
-  - real
-  - char
+  - boolean_title_case
+  - char_pascal
+  - integer_pascal
+  - real_pascal
+  - string_pascal   # TODO
```

Comment:
I also checked http://wiki.freepascal.org/Variables_and_Data_Types, ya, agree your point that 'Standard Pascal does not make provision for the string data type'. However, it also says 'most modern compilers do'. It's good to keep the string type.
Anyway, leaving a `TODO` here is not very useful 😛